behave/features/cla_in_scope.feature

```diff
@@ -5,6 +5,7 @@ Background: Start page
 
 @cla-in-scope
 Scenario: Select the in scope special education needs category
-#  GIVEN THAT I am on the scope diagnosis page
-#  WHEN I select categories Education -> Special Education needs
-#  THEN I am taken to the Legal aid is available for this type of problem page that is located on /legal-aid-available
\ No newline at end of file
+    Given I am on the scope diagnosis page
+    When I select the category Education
+    And the category Special Educational needs
```

Comment:
Should this 'and' be 'Then'? Then I select the category Special Educational needs?

Reply:
I would have gone with 'And then select the category Special Educational needs' or, 'And I select the category "Special Educational needs"' which would make the category a variable and make the line re useable.

Reply:
Ok, will look at updating this.

Reply:
I had a look and the two methods need to work differently. One button uses href to click through and the other (special educational needs) goes somewhere completely different than the href so can't really reuse the step anyway.